holidayextras/ui-toolkit
# UXUI-247 - Removing dev folder and moving to component specific less

#### What does this PR do?
- Removes the `dev` folder for each component as it's really not required
- Move each less file into the specific components code directory
- Updates `readme.md` of each component to remove the `ui-` prefix
- Improves legibility of docs by adding spacing after a demo-box

#### What unit or integration tests does this PR have?
none

#### What selenium tests does this PR have?
none

#### How should a developer review this?
- Take a look at the code it should all be fairly straightforward
- Manually test as below

#### How should this be manually tested?
- `npm install`
- `grunt docs`
- check that the components in the docs still look great.

#### Any background context you want to provide?
- the `dev` folder is created by the component generator, we will put in a separate pr to remove this from the generator as its not required.

#### What are the relevant tickets?
https://holidayextras.jira.com/browse/UXUI-247
#### What gif best describes how you feel about this work?
![clean and tidy](http://media.giphy.com/media/HIrYP4RI9DpLy/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [ ] I've checked there is appropriate unit, selenium and regression test coverage.
- [x] I've tested this cross-browser/device for visual changes.
- [x] I've manually checked this work against any dependent systems.
- [x] I've checked that there are no negative speed or performance impacts from my work.
- [ ] I've added appropriate tracking.

#### IP or Developer Review:

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
